go.mod

```diff
@@ -130,6 +130,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/monitor/armmonitor v0.11.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v5 v5.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.2.0
+	github.com/zcalusic/sysinfo v1.1.2
```

Comment:
This package is a lightweight utility that only uses the go library to grab e.g. kernel version from files
https://github.com/zcalusic/sysinfo/blob/v1.1.2/go.mod

Reply:
No longer using because of build errors in Windows. Just copied the two functions needed